features/include/pcl/features/impl/3dsc.hpp

```diff
@@ -157,6 +157,12 @@ pcl::ShapeContext3DEstimation<PointInT, PointNT, PointOutT>::computePoint (
   // Get origin normal
   // Use pre-computed normals
   normal = normals[minIndex].getNormalVector3fMap ();
+  if (!std::isfinite(normal[0]) || !std::isfinite(normal[1]) || !std::isfinite(normal[2]))
```

Comment:
```suggestion
  if(!pcl::isFinite(normal))
```

Might require an include?

Reply:
Or better, `isNormalFinite` from same header (common/point_tests.h)(because isFinite usually checks for x,y,z only)